holidays/countries/afghanistan.py

```diff
@@ -0,0 +1,108 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import InternationalHolidays, IslamicHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Afghanistan(HolidayBase, InternationalHolidays, IslamicHolidays):
+    """
+    https://en.wikipedia.org/wiki/Public_holidays_in_Afghanistan
+    https://www.timeanddate.com/holidays/afghanistan/
+    """
+
+    country = "AF"
+    default_language = "fa"
+    # %s (estimated).
+    estimated_label = tr("%s (برآورد شده)")
+    # %s (observed).
+    observed_label = tr("%s (مشاهده شده)")
+    # %s (observed, estimated).
+    observed_estimated_label = tr("%s (مشاهده شده، برآورد شده)")
+    supported_languages = ("en_US", "fa", "ps_AF")
+    # Afghanistan's regaining of full independence from British influence.
+    start_year = 1919
+
+    def __init__(self, *args, **kwargs):
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        if self._year >= 1989:
+            # Liberation Day.
+            self._add_holiday_feb_15(tr("روز آزادی"))
+
+        # Afghanistan Independence Day.
+        self._add_holiday_aug_19(tr("روز استقلال افغانستان"))
+
+        if self._year <= 1996 or 2001 <= self._year <= 2020:
+            # Nowruz.
+            self._add_holiday_mar_21(tr("نوروز"))
+
+        if self._year >= 1992:
+            # Mujahideen Victory Day.
+            self._add_holiday_apr_28(tr("روز پیروزی مجاهدین"))
+
+        if 1974 <= self._year <= 1996 or 2002 <= self._year <= 2021:
+            # International Workers' Day.
+            self._add_labor_day(tr("روز جهانی کارگر"))
+
+        if 1978 <= self._year <= 1988:
+            # Soviet Victory Day.
+            self._add_holiday_may_9(tr("روز پیروزی شوروی"))
+
+        if self._year >= 2022:
+            # American Withdrawal Day.
+            self._add_holiday_aug_31(tr("روز خروج آمریکایی‌ها"))
```

Comment:
```suggestion
            self._add_holiday_aug_31(tr("روز خروج آمریکایی ها"))
```
Replace U+200C with regular space. Or maybe it should be deleted, I'm not sure.